RELEASE_NOTES.rst

```diff
@@ -21,6 +21,113 @@
 
 .. towncrier release notes start
 
+Airflow 2.5.2 (2023-03-13)
+--------------------------
+
+Significant Changes
+^^^^^^^^^^^^^^^^^^^
+
+The date-time fields passed as API parameters or Params should be RFC3339-compliant (#29395)
+"""""""""""""""""""""""""""""""""""""""""""""""""""""""""""""""""""""""""""""""""""""""""""""
+
+  In case of API calls, it was possible that "+" passed as part of the date-time fields were not URL-encoded, and
+  such date-time fields could pass validation. Such date-time parameters should now be URL-encoded (as ``%2B``).
+
+  In case of parameters, we still allow IS8601-compliant date-time (so for example it is possible that
+  ' ' was used instead of ``T`` separating date from time and no timezone was specified) but we raise
+  deprecation warning. (#29395)
```

Comment:
```suggestion
In case of API calls, it was possible that "+" passed as part of the date-time fields were not URL-encoded, and
such date-time fields could pass validation. Such date-time parameters should now be URL-encoded (as ``%2B``).

In case of parameters, we still allow IS8601-compliant date-time (so for example it is possible that
' ' was used instead of ``T`` separating date from time and no timezone was specified) but we raise
deprecation warning.
```

Don't need these indented (not sure why recent releases are, PR incoming), nor the PR number duplicated.

Reply:
Indeed.

The output of:
```
towncrier build --version=${VERSION_WITHOUT_RC} --date=2023-03-14 --dir . --config newsfragments/config.toml
```
Is not `rst` formatted. So basically I end up editing and formatting this manually, to be consistent with previous releases.
This leave space for inconsistencies.

Same for the body of the changelog which comes from the following command, there is quite a few manual formatting operations here as well.
```
./dev/airflow-github changelog  v2-5-stable  v2-5-test
```

This is something I wanted to ask @ephraimbuddy, maybe there is a better way for building the release note I am not aware of.

Reply:
Fixed